.github/workflows/ci.yml

```diff
@@ -12,10 +12,12 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        ruby: [1.9, '2.0', 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, '3.0', head]
+        ruby: [1.9, '2.0', 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, '3.0', 3.1, 3.2, head]
     runs-on: ${{ matrix.os }}
+    env:
+      BUNDLE_WITHOUT: development
```

Comment:
Avoid installing development gems in CI. This will marginally speed up the build.